app/src/main/res/layout/fragment_setting.xml

```diff
@@ -80,7 +84,7 @@
         android:text="@string/reset_game_levels"
         android:textAllCaps="false"
         android:layout_margin="20dp"
-        android:layout_alignParentBottom="true" />
+        android:layout_below="@+id/RelativeLayoutSeekBarSoundEffects" />
```

Comment:
lets do some standard because we using different way to ids:
seekBarSoundEffectsRelativeLayout
seekBarSoundEffectsRLayout
seekBarSoundEffectsRL
?
same for all ids, lets be consist

Reply:
ok ill change all of them to "seekBarSoundEffectsRelativeLayout" on layouts